_guides/analytics/analytics_basics.md

```diff
@@ -0,0 +1,229 @@
+{% endhighlight %}
+
+Due to it’s simplicity,
+the `amp-pixel` tag can only include variables defined by the platform
+or that the AMP runtime can parse from the AMP page.
+The `amp-analytics` tag can include the same variables as `amp-pixel`,
+as well as uniquely defined variables inside the tag configuration. 
+
+Use the format `${varName}` in a request string for a page
+or platform-defined variable.
+The `amp-analytics` tag will replace the template with its actual value
+at the time of construction of the analytics request (see also
+[Variables supported in amp-analytics](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/analytics-vars.md)).
+
+In the following example, the page view request is sent to the URL,
+with additional data extracted from variable substitutions,
+some provided by the platform,
+some within the `amp-analytics` configuration itself:
+
+{% highlight html linenos %}
+<amp-analytics>
+<script type="application/json">
+{
+  "requests": {
+    "pageview":"https://example.com/analytics?url=${canonicalUrl}&title=${title}&acct=${account}&clientId=${clientId(cid-scope)}",
+  },
+  "vars": {
+    "account": "ABC123",
+  },
+  "triggers": {
+    "some-event": {
+      "on": "visible",
+      "request": "pageview",
+      "vars": {
+        "title": "My homepage",
+        "clientId": "user12345"
+      }
+    }
+  }  
+}
+</script>
+</amp-analytics>
+{% endhighlight %}
+
+In the above example,
+the variables, `account`, `title`, and `clientID` are defined
+in the `amp-analytics` configuration.
+The `canonicalUrl` variable isn’t defined in the configuration,
+so it's value gets substituted by the platform.
+
+**Important:** Variable substitution is flexible;
+you can have the same variables defined in different locations,
+and the AMP runtime will parse the values in this order of precedence
+(see [Variable substitution ordering](/docs/guides/analytics/deep_dive_analytics.html#variable-substitution-ordering)).
+
+## User identification
+
+Websites use cookies to track a user’s browsing history;
+cookies tell the browser that a user visited a web page.
+In AMP,
+pages can be served from either a publisher's website or the AMP Cache.
+The publisher’s website and the AMP Cache have different domains.
+For security reasons,
+browsers can (and often will) limit access to another domain’s cookies
+(see also
+[Tracking users across origins](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/cross-origin-tracking.md)).
+
+By default,
+AMP automatically creates a client ID to identify
+that a user has accessed an AMP page,
+regardless if that page is accessed from the publisher’s website
+or the AMP Cache.
+The AMP-generated client ID has a value of `"amp-"`
+followed by a random `base64` encoded string and remains the same
+for the same user if they visit again within one year.
+
+The client ID is always created by AMP
```

Comment:
AMP manages reading and writing of the client ID in all cases. This is particularly notable in the case when a page is served via a cache or otherwise shown outside the viewing context of the publisher's original site, as in this circumstance access to the publisher site's cookies is unavailable.

When an AMP page is served from a publisher's site, the client ID framework that AMP uses can be told about a fallback cookie to look for and use. In this case, the `cid-scope-cookie-fallback-name` argument of the client ID variable, which may appear as either `CLIENT_ID(cid-scope-cookie-fallback-name)` or `${clientId(cid-scope-cookie-fallback-name)}`, will be interpreted as a cookie name. If AMP finds that this cookie is set, then the client ID substitution will return the cookie's value. If the AMP finds that this cookie is not set, then AMP will generate a value of the form `"amp-" followed by a random base64 encoded string`.